.github/ISSUE_TEMPLATE/backport-request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Backport Request
+about: Any request for backport something specific
+title: "[Backport] Your title"
+labels: ''
+assignees: ''
+
+---
+
+**Original feature issue** (if present): **REPLACEME**
```

Comment:
- Title format for the section
- Move the description of what need to be describe in comment tag on the next line
- Ask for a PR number as reference?
```suggestion
#### Original feature issue
<!--- (if present): Describe the fearture --->
<!--- Is there an PR number for that feature? --->
```

Reply:
Done as desired